src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -113,6 +117,14 @@ export default class PatientViewMutationTable extends MutationTable<
                 ),
             download: (d: Mutation[]) =>
                 AlleleFreqColumnFormatter.getFrequency(d),
+            numericalFilter: (d: Mutation[], config: NumericalFilterConfig) =>
```

Comment:
I don't think we should couple a specific filter logic with the column definition. Are we going to add a `categoricalFilter` as well? I would instead define new `DataFilter`s (called `NumericalDataFilter`, `CategoricalDataFilter`, etc.) and handle all the logic inside a filter applier (say `NumericalDataFilterApplier`, `CategoricalFilterApplier`, etc.). By this way we wouldn't couple any filter specific logic with the column definition and also keep all data filters in one place. That would also simplify the cases where multiple data filters applied at once.

Here is an example implementation where we add a custom filter for the results view mutation mapper store: https://github.com/cBioPortal/cbioportal-frontend/blob/c0be78ff9eced5400308c82cd69cf142cf227bad/src/pages/resultsView/ResultsViewPageStore.ts#L3458

One problem with this approach is that we wouldn't be supporting generic numerical filters by default for the `LazyMobXTable`, but maybe we shouldn't worry about that for now.